Web/HTML/core-html-elements/update-images.md

```diff
@@ -70,6 +101,8 @@ Which of these are not a common web image format:
 ---
 ## Quiz
 
+### How much do you know about HTML `<img>` element?
```

Comment:
Ah Quiz questions have to have only one answer. 
Make one of these a revision question.